src/main/java/com/ausregistry/jtoolkit2/se/DomainRem.java

```diff
@@ -36,5 +36,33 @@ public DomainRem(String[] nameservers, String[] techContacts,
         super(AddRemType.REM, nameservers, techContacts, adminContacts,
                 billingContacts, statuses);
     }
-}
 
+    /**
+     * Remove associations from a domain.
+     *
+     * @param nameservers A  host as attribute with
+     * associated domain. To not remove any nameservers, this parameter
+     * must be null.
+     *
+     * @param techContacts A list of technical contact identifiers to
+     * disassociate from a domain. To not remove any technical contacts,
+     * this parameter must be null.
+     *
+     * @param adminContacts A list of admin contact identifiers to
+     * disassociate from a domain. To not remove any admin contacts,
+     * this parameter must be null.
+     *
+     * @param billingContacts A list of billing contact identifiers to
+     * disassociate from a domain. To not remove any billing contacts,
+     * this parameter must be null.
+     *
+     * @param statuses A list of statuses to remove from the domain. To
+     * not remove any statuses, this parameter must be null.
+     */
+    public DomainRem(Status[] statuses, String[] techContacts,
+        String[] adminContacts, String[] billingContacts, Host[] nameservers) {
```

Comment:
Same as above, please swap the first and last parameters (see below):
```java
    public DomainRem(Host[] nameservers, String[] techContacts,
        String[] adminContacts, String[] billingContacts, Status[] statuses) {
```

Reply:
Yes ,Bin you are correct that its preferred to keep the parameter list in same order. But if keep the order in DomainRem as you said and  if  we are not adding/removing name servers in domain update command then this value will be passed as null (test case will fail because there will be 2 constructor which would be ambiguous  as we are passing null for parameter which was differentiator ) .
Now if think to add other argument and try to use we cant because super() is called . 
We cant even modify the old constructor to accept host[] instead of String[] because that will not be backward compatible if we do so .